src/App.css

```diff
@@ -1,5 +1,39 @@
+body {
```

Comment:
I'm wondering why you need to apply styles to both the `body` element and the `.App` element. If you apply the background color here to the `.App` element, can we delete the CSS for `body`?

Reply:
Yeah, I didn't need to. Just updated it!